src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card">
+      <div class="card__img">
+        <img
+          src="./images/imac.jpeg"
+          alt="card"
+          width="160"
+          height="134"
+        >
+      </div>
+
+      <div class="card__content">
+        <div class="content">
+          <p class="title">Моноблок APPLE A1419 iMac  27" Retina 5K (MNED2UA/A)
+          </p>
+          <span class="code-card">Код товара: 195434</span>
+        </div>
+        <div class="rate">
+          <div class="stars stars--4">
+            <div class="stars__star is-active"></div>
+            <div class="stars__star is-active"></div>
+            <div class="stars__star is-active"></div>
+            <div class="stars__star is-active"></div>
+            <div class="stars__star"></div>
```

Comment:
используй стили для звезд с прошлого задания
```suggestion
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
```